### PR TITLE
add method and test webinar report

### PR DESCRIPTION
**Pull request by sharinetmc**
Add get_webinar_report method to zoom connector

### CODE REVIEW

File: parsons/zoom/zoom.py [hunk 1]
```diff
@@ -40,7 +40,7 @@ def __init__(self, account_id=None, client_id=None, client_secret=None):
             authorization_kwargs={"account_id": self.account_id},
         )
 
-    def _get_request(self, endpoint, data_key, params=None, **kwargs):
+    def _get_request(self, endpoint, data_key=None, params=None, **kwargs):
```
Comment (IanRFerguson) [inline parsons/zoom/zoom.py:43]: what's the rationale behind defaulting this to `None` type?

**Review by KasiaHinkson — APPROVED** [review]
LGTM

**Review by IanRFerguson — APPROVED** [review]
It looks like there's a formatting check with `ruff` that's failing - I would recommend looking into it to get it resolved but not existential imo

File: parsons/zoom/zoom.py (hunk 1 above)
Comment (sharinetmc) [reply parsons/zoom/zoom.py:43]: This method that I have returns data that doesn't need to be parsed further using a key.

Comment (sharinetmc) [reply parsons/zoom/zoom.py:43]: But I guess it doesn't necessarily need to be a default value

**Pull request merged**